CorsixTH/Lua/app.lua

```diff
@@ -1827,15 +1827,16 @@ end
 function App:getSystemInfo()
   local compile_opts = TH.GetCompileOptions()
   local comp_details = {}
+  local date = os.date("%Y-%m-%d %H:%M:%S") -- Use ISO 8601 standard
   for key, value in pairs(compile_opts) do
     table.insert(comp_details, key .. ": " .. tostring(value))
   end
   table.sort(comp_details)
-  local compiled = string.format("Compiled with %s\nSDL renderer: %s\n",
-    table.concat(comp_details, ", "), self.video:getRendererDetails())
+  local compiled = string.format("Gamelog generated on %s\nCompiled with %s\nSDL renderer: %s\n",
```

Comment:
Now potentially out of scope of the function.. maybe be isolated to the gamelog functions instead (world:dumpgamelog and app:initgamelog).